packages/protocol/contracts/layer1/preconf/impl/PreconfWhitelist.sol

```diff
@@ -200,30 +134,49 @@ contract PreconfWhitelist is EssentialContract, IPreconfWhitelist, IProposerChec
         view
         returns (bool)
     {
-        if (_proposer == address(0)) return false;
-        OperatorInfo memory info = operators[_proposer];
-        if (_epochTimestamp < info.activeSince) {
-            return false;
-        } else if (info.inactiveSince != 0 && _epochTimestamp >= info.inactiveSince) {
```

Comment:
is this original code buggy? shall 'true' be returned if `info.inactiveSince != 0 && _epochTimestamp >= info.inactiveSince` is true?